bindings/python/pydeck/docs/layer.md

```diff
@@ -9,30 +9,69 @@ class pydeck.Layer(
     type,
     data,
     id=None,
-    get_position='[lng,lat]',
+    get_position='[lng, lat]',
     **kwargs)
 ```
 
-Constructs a geospatial layer for plotting.
+Constructs a visualization layer from data for rendering on a map.
 
 A catalog of available layers is viewable [here](https://github.com/uber/deck.gl/tree/master/docs/layers#deckgl-layer-catalog-overview).
+
 Note that this is Javascript documentation and parameters in pydeck usually will be snake-cased according to Python convention.
 
+
+
 #### Parameters
 
 `type` : `str`
-    Type of layer to render, e.g., `HexagonLayer`. See the layer catalog above.
+
+Type of layer to render, e.g., `HexagonLayer`. See the [layer catalog](https://github.com/uber/deck.gl/tree/master/docs/layers#deckgl-layer-catalog-overview).
+    
 `data` : `str` or `list` of `dict` or `pandas.DataFrame`
-    A URL of data to load in, a list of dictionaries,
+
+A URL of data to load in, a list of dictionaries, or a pandas.DataFrame to load into the visualization
+
 `id` : `str`, default `None`
-    Unique name for the layer. Will autopopulate with a UUID if no ID is provided.
-`get_position` : `str`, default `'[lng,lat]'`
-    Name of position field, with some expression parsing, e.g., `'[lng,lat]'` is provided, the position field will be assumed to be the common case where a flat file has separate columns `lat` and `lng`.
+
+Unique name for the layer. Will autopopulate with a UUID if no ID is provided.
+
+`get_position` : `str`, default `'[lng, lat]'`
```

Comment:
Why is `get_position` documented but not others? What about the layers that do not have `getPosition` props? Shouldn't we point to the JavaScript layer docs and explain how it's translated to python?

Reply:
Addressed in update

Reply:
get_position should not be called out period. You need to explain how accessors are handled in general.